embd.go

```diff
@@ -176,12 +194,12 @@ package {{.Pkg}}
 {{range .Files}}
 // {{.VarName}} contains contents of "{{.Path}}" file.
 var {{.VarName}} = []byte("{{range .DataFragments}}{{.}}{{end}}")
-{{end}}`[1:]
+{{end}}
 
-/*
-var dirTemplate = `
-var {NAME} = map[string][]byte{
-{ENTRIES}
+{{range $index, $element := .Dirs}}
+var Dir_{{$index}} = map[string][]byte{
```

Comment:
1. Please don't use {{$index}}, instead sanitize the directory names (first simplified with filepath.Base) with below function:

        func normalize(path string) string {
            return Normalize.ReplaceAllString("_"+filepath.Base(path), "_")
        }

    then build structs like below:

        var Dir_mydir = struct {
            File_myfile []byte
            File_otherfile []byte
        }{
            []byte("....."),
            []byte("......"),
        }

2. Use the above function to build VarName too (`"File" + normalize(filepath.Base(path))`).
3. Please keep the file built by template already gofmted as much as possible (in the above example, you can leave field types unaligned vertically, I don't have idea how to do that easily here, unfortunately)